runtime/lua/vim/treesitter.lua

```diff
@@ -275,7 +278,7 @@ end
 ---@param opts table Optional keyword arguments:
 ---             - ignore_injections boolean Ignore injected languages (default true)
 ---
----@returns (table) Named node under the cursor
+---@return userdata |tsnode| under the cursor
```

Comment:
I rather like this way of liking to other help file than the `... see |LangTree|` do you think you can do that everywhere ?

Reply:
No, because the `LanguageTree` is a type that is interpreted by lua-language-server, and `|LanguageTree|` would break that.